src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -339,4 +340,105 @@ default GameModeData getGameModeData() {
+     * @param diameter The new diameter of the border.
+     */
+    default void setWorldBorderDiameter(double diameter) {
+        getWorldBorder().setDiameter(diameter);
+    }
+
+    /**
+     * Changes the diameter of the player's world border over a given amount of
+     * time.
+     * 
+     * @param diameter The new diameter of the border.
+     * @param time The amount of time it will take for the border to reach the
+     * new diameter.
+     */
+    default void setWorldBorderDiameter(double diameter, long time) {
+        getWorldBorder().setDiameter(diameter, time);
+    }
+
+    /**
+     * Sets the diameter of the player's world border to the starting diameter
+     * initially and changes to the end diameter over a specified amount of 
+     * time.
+     * 
+     * @param startDiameter The starting diameter to use initially.
+     * @param endDiameter The final diameter to change to eventually.
+     * @param time The amount of time to change for.
+     */
+    default void setWorldBorderDiameter(double startDiameter, double endDiameter, long time) {
+        getWorldBorder().setDiameter(startDiameter, endDiameter, time);
+    }
+
+    /**
+     * Sets the amount of damage to give to the player if they leave their world
+     * border.
+     * 
+     * @param damage The amount of damage to give to the player.
+     */
+    default void setWorldBorderDamageAmount(double damage) {
+        getWorldBorder().setDamageAmount(damage);
+    }
+
+    /**
+     * Sets the damage threshold of the player's world border. This determines 
+     * how far the player has to go outside of the border before being damaged.
+     * 
+     * @param distance The distance before the player gets damaged.
+     */
+    default void setWorldBorderDamageThreshold(double distance) {
+        getWorldBorder().setDamageThreshold(distance);
+    }
+
+    /**
+     * Sets the warning distance of the player's world border. This determines 
+     * how far the player has to be near to their border before getting the 
+     * warning. This is only effective when the border is contracting.
+     * 
+     * @param distance The warning distance.
+     */
+    default void setWorldBorderWarningDistance(int distance) {
+        getWorldBorder().setWarningDistance(distance);
+    }
+
+    /**
+     * Sets the warning time of the player's world border. This determines when 
+     * the warning is displayed depending on how long the border will take to 
+     * reach the player. This is only effective when the border is contracting.
+     * 
+     * @param time The warning time.
+     */
+    default void setWorldBorderWarningTime(int time) {
+        getWorldBorder().setWarningTime(time);
+    }
+
+    /**
+     * Sets the {@link WorldBorder} for this player to the given world border.
+     * 
+     * @param border The world border to be used.
+     * @param cause The cause of the border's change.
+     */
+    void setWorldBorder(WorldBorder border, Cause cause);
```

Comment:
Is there actually a way to create a new `WorldBorder`? As far as I can see you can only get the `WorldBorder` from worlds, but you can't create new ones.